# Bioinformatics

* [tcga-code-tables](https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables)
* [诺禾信息平台](https://disease.novogene.org/homepage/) - 诺禾信息平台
* [HPO](http://human-phenotype-ontology.github.io/) - Human Phenotype Ontology 人类表型数据库
* [NGS-Workflows](https://jmazz.me/blog/NGS-Workflows)
* [gkno](http://gkno.me) - A bioinformatics pipeline launcher
* [Genenteck](https://www.gene.com/stories/) - 一个博客
* [oncotree](http://oncotree.mskcc.org/#/home) - 肿瘤分类
* [Cancer today](http://gco.iarc.fr/today/home) - 患病率查询
* [GeneMANIA](https://genemania.org/) - GeneMANIA helps you predict the function of your favourite genes and gene set
### pipeline
* [Nextflow](https://www.nextflow.io/docs/latest/index.html)
* [Snakemake](https://snakemake.readthedocs.io/en/stable/index.html)

### 肿瘤数据库

* [GENIE](http://www.aacr.org/research/research/pages/aacr-project-genie.aspx)
* [phangorn](https://github.com/KlausVigo/phangorn) - 肿瘤进化算法

### 倍性与纯度

* [sequenza](https://bitbucket.org/sequenza_tools/sequenza/)
* [ABSOLUTE](http://archive.broadinstitute.org/cancer/cga/absolute)  - hbsun

### 基因查询
* [CDC](https://www.cdc.gov/)
* [突变信息转换](http://bioinformatics.mdanderson.org/transvarweb/)
* [Gencode](http://www.gencodegenes.org/)
* [Ensemble](http://www.ensembl.org/index.html)
* [Uniprob](http://www.uniprot.org/)
* [Genecards](http://www.genecards.org/)
* [EMBL-EBI](https://www.ebi.ac.uk/)
* [UCSC](http://genome.ucsc.edu/)
* [Genome Reference Consortium](https://www.ncbi.nlm.nih.gov/grc)
* [HGNC](http://www.genenames.org/) - 人类基因组命名委员会
* [ENCODE](https://www.encodeproject.org/)
* [cbioportal](http://www.cbioportal.org/) -  **visualization, analysis and download of large-scale cancer genomics data sets**
* [awesome-bio-datasets](https://github.com/OpenGene/awesome-bio-datasets)
* [pharmgkb](https://www.pharmgkb.org/) - 药物基因组
* [DrugBank](https://www.drugbank.ca/) - 药物数据库
* [DGV](http://dgv.tcag.ca/dgv/app/home) - 收录了健康人群中序列长度大于50bp的结构变异。

### 人群平率
* [ExAC](http://exac.broadinstitute.org/)
* [ESP6500](http://evs.gs.washington.edu/EVS/)
* [1000 Genome](http://www.internationalgenome.org/home)
* [GenomeAD](http://gnomad.broadinstitute.org/) - 最全人群频率数据库
* [awesome-bio-datasets](https://github.com/OpenGene/awesome-bio-datasets)
* [cngb](https://db.cngb.org/pird/home/) - 华大数据库
* [Orphanet](http://www.orpha.net/consor/cgi-bin/index.php?lng=EN) - 罕见病数据库以及孤儿药物

### 蛋白数据库
* [Uniprot](http://www.uniprot.org/)
* [Pfam](http://pfam.xfam.org/)
* [PBD](http://www.rcsb.org/pdb/home/home.do)
* [HPA](http://www.proteinatlas.org/) - Human Protein Atlas 数据库

### 注释工具
* [annovar](http://annovar.openbioinformatics.org/en/latest/)
* [Oncotator](http://portals.broadinstitute.org/oncotator/)
* [HGVS](http://varnomen.hgvs.org/) - 基因组注释说明
* [snpeff](http://snpeff.sourceforge.net/index.html) - Genomic variant annotations and functional effect prediction toolbox. 
* [Genomic variant annotations and functional effect prediction toolbox. ](https://genomemedicine.biomedcentral.com/articles/10.1186/gm543)
* [trackViewer](https://bioconductor.org/packages/release/bioc/html/trackViewer.html)
* [mygene](http://docs.mygene.info/en/latest/index.html) - 基因ID转换
### Cancer database
* [CIVIC](https://civic.genome.wustl.edu/home)
* [COSMIC](http://cancer.sanger.ac.uk/cosmic)
* [OncoKB](http://oncokb.org)



### CNV & SNP

* [ascat](https://github.com/Crick-CancerGenomics/ascat) - Allele-specific copy number analysis of tumors
* [GATK](https://software.broadinstitute.org/gatk/)
* [VarScan](http://varscan.sourceforge.net/)
* [strelka](https://github.com/Illumina/strelka) -  fast and accurate small variant caller
* [samtools](http://www.htslib.org/doc/#manual-pages)
* [freebayes](https://github.com/ekg/freebayes) - a haplotype-based variant detector
* [bedtools](http://bedtools.readthedocs.io/en/latest/) -  a powerful toolset for genome arithmetic
* [HMMcopy](http://www.bioconductor.org/packages/release/bioc/html/HMMcopy.html) 
* [facets](https://sites.google.com/site/mskfacets/) - CNV for WES
* [CNVkit](http://cnvkit.readthedocs.io/en/stable/pipeline.html) - CNV分析，基于on-target和off-target
* [CopywriteR](http://www.bioconductor.org/packages/release/bioc/html/CopywriteR.html) - CNV分析，基于off-target
* [DNAcopy](http://www.bioconductor.org/packages/release/bioc/html/DNAcopy.html) - CNV分析
* [CONTRA](http://contra-cnv.sourceforge.net/) - 目标区域CNV分析
* [copyCat](https://github.com/chrisamiller/copycat) - somatic copy number aberrations
* [pindel](https://github.com/genome/pindel) - detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications 
* [breakdancer](http://breakdancer.sourceforge.net/) - genome-wide detection of structural variants from next generation paired-end sequencing reads
* [sequenza](http://www.cbs.dtu.dk/biotools/sequenza/) - estimate cancer cellularity, ploidy, genome wide copy number profile and infer for mutated alleles.

* [lumpy](https://github.com/arq5x/lumpy-sv) - A probabilistic framework for structural variant discovery
* [manta](https://github.com/Illumina/manta) - Structural variant and indel caller
* [MutScan](https://github.com/OpenGene/MutScan) - Detect and visualize target mutations by scanning FastQ files directly
* [ExomeCNV](https://secure.genome.ucla.edu/index.php/ExomeCNV_User_Guide) - CNV (Copy-Number Variants) and LOH (Loss of Heterozygosity) from exome sequencing data

### Gene fusion
* [GeneFuse](https://github.com/OpenGene/GeneFuse) - Gene fusion detection and visualization

### MSI
* [msisensor](https://github.com/ding-lab/msisensor)
* [VisualMSI](https://github.com/OpenGene/VisualMSI)

### RNA
* [rna-seq-strand](https://github.com/igordot/genomics/blob/master/notes/rna-seq-strand.md) - RNA分析链向选择
* [Hisat2](http://ccb.jhu.edu/software/hisat2/index.shtml) - graph-based alignment of next generation sequencing reads to a population of genomes
* [stringtie](http://www.ccb.jhu.edu/software/stringtie/) - Transcript assembly and quantification for RNA-Seq
* [gffread](http://ccb.jhu.edu/software/stringtie/gff.shtml#gffread) - 通过gff文件提取转录本fasta序列
* [Xfam](http://rfam.xfam.org/) - RNA数据库
* [featureCounts](http://bioinf.wehi.edu.au/featureCounts/) - map到基因的reads统计
* [htseq](http://htseq.readthedocs.io/en/release_0.9.1/) - map到基因的reads统计htseq-count
* [miRBase](http://www.mirbase.org/) - miRNA数据库
* [STAR](https://github.com/alexdobin/STAR) - RNAseq mapping tools
* [STAR-Fusion](https://github.com/STAR-Fusion/STAR-Fusion/wiki) - 基因融合分析工具
* [Trinity_CTAT](https://github.com/NCIP/Trinity_CTAT/wiki) - Trinity Cancer Transcriptome Analysis Toolkit 
* [Trinity](https://github.com/trinityrnaseq/trinityrnaseq/wiki) - RNA-Seq De novo Assembly
* [Cufflinks](http://cole-trapnell-lab.github.io/cufflinks/) - Transcriptome assembly and differential expression analysis for RNA-Seq
* [GimmeMotifs](http://gimmemotifs.readthedocs.io/en/master/) - motif analysis
* [gkmsvm](http://www.beerlab.org/gkmsvm/) - Enhanced Regulatory Sequence Prediction Using Gapped k-mer Features
* [meme](http://meme-suite.org/index.html) - Motif-based sequence analysis tools
* [rnacocktail](https://bioinform.github.io/rnacocktail/) - RNA分析流程 DOI: 10.1038/s41467-017-00050-4
* [MSigDB](http://software.broadinstitute.org/gsea/msigdb) -  a collection of annotated gene sets for use with GSEA software
* [rbsurv](http://www.bioconductor.org/packages/release/bioc/html/rbsurv.html) - This package selects genes associated with survival. 获取与生存时间相关的基因
* [WGCNA](https://cran.r-project.org/web/packages/WGCNA/index.html) - WGCNA其译为加权基因共表达网络分析。该分析方法旨在寻找协同表达的基因模块(module)，并探索基因module与关注的表型之间的关联关系，以及网络中的核心基因。http://www.stat.wisc.edu/~yandell/statgen/ucla/WGCNA/wgcna.html
* [CIBERSORT ](https://cibersort.stanford.edu/) - to provide an estimation of the abundances of member cell types in a mixed cell population, using gene expression data.

### 基因组组装
* [platanus](http://platanus.bio.titech.ac.jp)  复杂基因组组装
* [allpath](http://software.broadinstitute.org/allpaths-lg/blog/)  
* [MECAT](https://github.com/xiaochuanle/MECAT) - 三代组装软件
* [megahit](https://github.com/voutcn/megahit) - meta组装，速度快内存消耗少
### QC
* [trimmomatic](http://www.usadellab.org/cms/index.php?page=trimmomatic) - 去接头软件
* [fastp](https://github.com/OpenGene/fastp) - 质控软件
* [AfterQC](https://github.com/OpenGene/AfterQC) - 质控软件
* [MultiQC](https://github.com/ewels/MultiQC) - 整合QC质控结果

### Mapping
* [BBmap](https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbmap-guide/)
* [GSNAP](http://research-pub.gene.com/gmap/) - Genomic Short-read Nucleotide Alignment Program

### Call mutation
* [MuSE](http://bioinformatics.mdanderson.org/main/MuSE)  MD Anderson
* [SomaticSniper](http://gmt.genome.wustl.edu/packages/somatic-sniper/)
* [strelka](https://github.com/Illumina/strelka)

### 基因预测
* [AUGUSTUS](http://bioinf.uni-greifswald.de/augustus/) - AUGUSTUS is a program that predicts genes in eukaryotic genomic sequences
* [msisensor](https://github.com/ding-lab/msisensor) - microsatellite instability detection using paired tumor-normal 微卫星不稳定
* [TransDecoder](https://github.com/TransDecoder/TransDecoder) - CDS预测

### TCR

* [MIGEC](http://migec.readthedocs.io/en/latest/index.html) - Molecular Identifier Guided Error Correction pipeline
* [igblast](https://www.ncbi.nlm.nih.gov/igblast/) - NCBI开发的用于比对TCR的工具

### 进化

* [CITUP](https://github.com/sfu-compbio/citup) - Clonality Inference in Multiple Tumor Samples using Phylogeny
* [deconstructSigs](https://github.com/raerose01/deconstructSigs) - R语言 signature分析
* [LICHeE](http://viq854.github.io/lichee/) - Fast and scalable inference of multi-sample cancer lineages

* [ExpressBetaDiversity](https://github.com/dparks1134/ExpressBetaDiversity) - 可基于tree计算beta多样性

### 常用工具

* [STRING](https://string-db.org/cgi/input.pl?UserId=1hcUYxDKX0aK&sessionId=xykqPVdtcqrE&input_page_show_search=off) - 蛋白互作
* [SetRank](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1571-6) - 富集分析
* [GSVA](http://bioconductor.org/packages/release/bioc/html/GSVA.html) - 富集分析，可以进行ssGSEA分析
* [PureCN](http://bioconductor.org/packages/release/bioc/html/PureCN.html) - This package estimates tumor purity, copy number, and loss of heterozygosity (LOH), and classifies single nucleotide variants (SNVs) by somatic status and clonality.
* [bam-readcount](https://github.com/genome/bam-readcount) - 位点深度统计
* [sciclone](https://github.com/genome/sciclone) - 克隆进化分析
* [pyclone](http://compbio.bccrc.ca/software/pyclone/) - 克隆进化分析
* [Wgsim](https://github.com/lh3/wgsim) - 数据模拟
* [bamgineer](https://github.com/pughlab/bamgineer) - 数据模拟
* [btctools](https://github.com/josephhughes/btctools) - 计算ds/dn
* [vcf2bed](http://bedops.readthedocs.io/en/latest/content/reference/file-management/conversion/vcf2bed.html)
* [deconstructSigs](https://github.com/raerose01/deconstructSigs) - signature分析
* [ESTIMATE](http://bioinformatics.mdanderson.org/main/ESTIMATE:Overview) - is a tool for predicting tumor purity, and the presence of infiltrating stromal/immune cells in tumor tissues using gene expression data. 
* [slicer](https://github.com/OpenGene/slicer) - Slice a text file (like FastQ) to smaller files by lines, with gzip supported
* [Vt](https://genome.sph.umich.edu/wiki/Vt) - a variant tool set that discovers short variants from Next Generation Sequencing data.

* [hgvs](https://github.com/biocommons/hgvs) - 解析hgvs


### 网站收集
* [CCB](https://ccb.jhu.edu/software.shtml) - 约翰霍普金斯大学计算生物学研究中心(RNA分析工具、meta分析工具)
* [quinlanlab](http://quinlanlab.org/#portfolioModal6) - Quinlan实验室，分析工具
* [raphael lab](http://compbio.cs.brown.edu/software) - raphael lab 实验室
* [学习网站](https://www.plob.org/) 
* [cgdsr](https://cran.r-project.org/web/packages/cgdsr/index.html) - TCGA临床数据下载 
* [cbioportal](http://www.cbioportal.org/) - TCGA临床数据下载,数据可视化MutationMapper、OncoPrinter
* [perl计算序列相似性](http://www.perlmonks.org/?node_id=500235) - 计算序列的相似性perl
* [alignment_tools](https://en.m.wikipedia.org/wiki/List_of_sequence_alignment_software) - alignment工具总结
* [intogen](https://www.intogen.org/search) - 可视化
* [homer](http://homer.ucsd.edu/homer/) - motif

### 微生物
* [kraken](http://ccb.jhu.edu/software/kraken/MANUAL.html) - Kraken taxonomic sequence classification system
* [microbiology](http://microbiology.se/) - 有一些分析软件

### 甲基化
* [ChAMP](http://www.bioconductor.org/packages/release/bioc/vignettes/ChAMP/inst/doc/ChAMP.html) - 甲基化分析
* [ChAMP](https://blog.csdn.net/joshua_hit/article/details/54982018) - 甲基化博客

### 驱动基因
* [mutsigcv](http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/mutsigcv) - 驱动基因
* [OncodriveFML](https://bitbucket.org/bbglab/oncodrivefml)
